MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -563,6 +554,31 @@
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
+		C51923B41EA55C5E002AF9E1 /* Extensions */ = {
+			isa = PBXGroup;
+			children = (
+				351BEC081E5BCC72006FE110 /* Bundle.swift */,
+				351BEC0A1E5BCC72006FE110 /* Directions.swift */,
+				351BEC041E5BCC6C006FE110 /* ManeuverDirection.swift */,
+				351BEBEB1E5BCC63006FE110 /* String.swift */,
+				351BEBF01E5BCC63006FE110 /* UIView.swift */,
+			);
+			name = Extensions;
+			sourceTree = "<group>";
+		};
+		C51923B51EA55CD4002AF9E1 /* UI Style */ = {
+			isa = PBXGroup;
+			children = (
+				351BEC091E5BCC72006FE110 /* DashedLineView.swift */,
+				351BEC031E5BCC6C006FE110 /* LaneArrowView.swift */,
+				351BEBEC1E5BCC63006FE110 /* StyleButton.swift */,
+				351BEBED1E5BCC63006FE110 /* StyleKitArrows.swift */,
+				351BEBEE1E5BCC63006FE110 /* StyleLabel.swift */,
+				351BEBEF1E5BCC63006FE110 /* TurnArrowView.swift */,
+			);
+			name = "UI Style";
```

Comment:
How about “Views”? This group contains views (and some classes that implement custom drawing of views) that aren’t part of the route table.